exporter/azure_monitor/common/common.go

```diff
@@ -52,3 +53,12 @@ type RemoteDependency struct {
 	Ver int				`json:"ver"`
 	Type string			`json:"type"`
 }
+
+type Request struct {
+	Name string 		`json:"name"`
+	Id string 			`json:"id"`

```

Comment:
suggestion: pay attention to the formatting/details

Reply:
The spacing is fine on my local machine, but I will try replacing tabs with spaces so this shows up cleaner.